mne_bids/read.py

```diff
@@ -383,14 +394,32 @@ def read_raw_bids(bids_fname, bids_root, extra_params=None,
         # Get coordinate frames that electrode coordinates are in
         if kind == "meg":
             coord_frame = coordsystem_json['MEGCoordinateSystem']
+            coord_unit = coordsystem_json['MEGCoordinateUnits']
+        elif kind == 'eeg':
+            coord_frame = coordsystem_json['EEGCoordinateSystem']
+            coord_unit = coordsystem_json['EEGCoordinateUnits']
+
+            if coord_frame.lower() not in ['captrak', 'besa']:
+                warn("Defaulting to 'head' coordinate frame for "
+                     "EEG montage because mne-bids does not "
+                     "support {} coordinate frame yet.".format(coord_frame))
+            coord_frame = 'head'
         elif kind == "ieeg":
-            coord_frame = coordsystem_json['iEEGCoordinateSystem']
+            coord_frame = coordsystem_json['iEEGCoordinateSystem'].lower()
```

Comment:
why the .lower()? it's not done for EEG or MEG

Reply:
I think the mne-python coordinate frames are all lower case, so I'll adjust for EEG and MEG.